<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-1SMJ8JH" failures="1" tests="4" name="Test" time="50.844" errors="0" timestamp="2023-12-27T00:01:53 IST">
  <testcase classname="qa.testcases.Login_Test" name="login_Test" time="2.614"/>
  <testcase classname="qa.testcases.User_Test" name="openUserPage" time="3.195"/>
  <testcase classname="qa.testcases.User_Test" name="addUserDetails" time="13.863"/>
  <testcase classname="qa.testcases.User_Test" name="validateUser" time="20.473">
    <failure type="java.lang.AssertionError" message="Employee name mismatch expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Employee name mismatch expected [true] but found [false]
at qa.testcases.User_Test.validateUser(User_Test.java:129)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- validateUser -->
  <testcase classname="qa.base.BaseTest" name="@AfterTest tearDown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest tearDown -->
  <testcase classname="qa.base.BaseTest" name="@BeforeTest initialization" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest initialization -->
</testsuite> <!-- Test -->
